ci/eval/default.nix

```diff
@@ -261,6 +261,11 @@ let
           --slurpfile after ${afterResultDir}/outpaths.json \
           > $out/changed-paths.json
 
+        jq -r '
+        "## Added\n" + (.attrdiff.added | map("- \(. )") | join("\n")) + "\n" +
+        "## Removed\n" + (.attrdiff.removed | map("- \(. )") | join("\n")) + "\n" +
+        "## Changed\n" + (.attrdiff.changed | map("- \(. )") | join("\n")) + "\n"
```

Comment:
```suggestion
        "## Added " + (.attrdiff.added | length | tostring) + "\n" + (.attrdiff.added | map("- \(. )") | join("\n")) + "\n" +
        "## Removed " + (.attrdiff.removed | length | tostring) + "\n" + (.attrdiff.removed | map("- \(. )") | join("\n")) + "\n" +
        "## Changed " + (.attrdiff.changed | length | tostring) + "\n" + (.attrdiff.changed | map("- \(. )") | join("\n")) + "\n"
```

How about this, will produce something like:

---

## Added 0

## Removed 0

## Changed 25
- hello
- tests.checkpointBuildTools
- tests.config.allowPkgsInPermittedInsecurePackages
- tests.devShellTools.nixos
- tests.devShellTools.unstructuredDerivationInputEnv
- tests.devShellTools.valueToString
- tests.hardeningFlags.stackClashProtectionExplicitDisabled
- tests.hardeningFlags.stackClashProtectionExplicitEnabled
- tests.hardeningFlags.stackClashProtectionStdenvUnsupp
- tests.hardeningFlags-clang.stackClashProtectionExplicitDisabled
- tests.hardeningFlags-clang.stackClashProtectionExplicitEnabled
- tests.hardeningFlags-clang.stackClashProtectionStdenvUnsupp
- tests.hardeningFlags-gcc.stackClashProtectionExplicitDisabled
- tests.hardeningFlags-gcc.stackClashProtectionExplicitEnabled
- tests.hardeningFlags-gcc.stackClashProtectionStdenvUnsupp
- tests.pkgs-lib
- tests.testers.nixosTest-example
- tests.testers.runNixOSTest-example
- tests.testers.testBuildFailure.helloDoesNotFail
- tests.trivial-builders.concat
- tests.trivial-builders.linkFarm
- tests.trivial-builders.references
- tests.trivial-builders.writeClosure-union
- tests.trivial-builders.writeShellApplication
- tests.trivial-builders.writeStringReferencesToFile